cloud/aws/templates/aws_oidc/secrets.tf

```diff
@@ -79,7 +79,7 @@ resource "aws_secretsmanager_secret_version" "postgres_password_secret_version"
 
 # Create a random generated password to use for app_secret_key.
 resource "random_password" "app_secret_key" {
-  length           = 16
+  length           = var.random_password_length
```

Comment:
Is there a way to error if this is smaller than the default?

Reply:
Yeah! I can add a validation, but we can do that once we update this default to the higher one.